<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWnd</class>
 <widget class="QWidget" name="TaskWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>693</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>112</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
				background-color: #52baff;
             font-family: Arial;        /* 字体 */
             font-size: 20px;           /* 字体大小 */
             font-weight: bold;         /* 字体粗细 */
}</string>
             </property>
             <property name="text">
              <string>设置定时提醒</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="midLineWidth">
                <number>1</number>
               </property>
               <property name="text">
                <string>小时</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>37</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
               </property>
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
				background-color: #52baff;
             font-family: Arial;        /* 字体 */
             font-size: 20px;           /* 字体大小 */
             font-weight: bold;         /* 字体粗细 */
}</string>
          </property>
          <property name="text">
           <string>紧急且重要</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
				background-color: #52baff;
             font-family: Arial;        /* 字体 */
             font-size: 20px;           /* 字体大小 */
             font-weight: bold;         /* 字体粗细 */
}</string>
          </property>
          <property name="text">
           <string>不紧急重要</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_2"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="horizontalFrame_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>112</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>37</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
             </property>
             <property name="text">
              <string>进度查看</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
				background-color: #52baff;
             font-family: Arial;        /* 字体 */
             font-size: 20px;           /* 字体大小 */
             font-weight: bold;         /* 字体粗细 */
}</string>
          </property>
          <property name="text">
           <string>紧急但不重要</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>190</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
				background-color: #52baff;
             font-family: Arial;        /* 字体 */
             font-size: 20px;           /* 字体大小 */
             font-weight: bold;         /* 字体粗细 */
}</string>
          </property>
          <property name="text">
           <string>不紧急也不重要</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_3"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="styleSheet">
             <string notr="true">            QPushButton {
                background-color: #52baff;
                color: #000000;
                padding: 10px;
                border-radius: 5px;
         font-family: Arial;        /* 字体 */
             font-size: 16px;           /* 字体大小 */
			font-weight:bold;
            }
            QPushButton:hover {
                background-color: #52bacc;
            }</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
